libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/speedlimit/SpeedLimitWidget.kt

```diff
@@ -26,68 +28,87 @@ class SpeedLimitWidget(
     /**
      * The horizontal margin of the widget relative to the map.
      */
-    marginX: Float = 26f,
+    marginX: Float = 14f,
     /**
      * The vertical margin of the widget relative to the map.
      */
-    marginY: Float = 50f,
-) : BitmapWidget(drawSpeedLimitSign(speedLimit = null, speed = 0), position, marginX, marginY) {
+    marginY: Float = 30f,
+    /**
+     * The initial sign format to use before speed limit info is available.
+     */
+    initialSignFormat: SpeedLimitSign = SpeedLimitSign.MUTCD,
```

Comment:
This seems like it should come from the device by default 🤔

Reply:
Actually, it would be better to not show the speed limit sign because it will likely change after [SpeedLimit.speedLimitSign] comes in

Reply:
Speed limit widget shows not only the speed limit, but also the current speed. That's why we have to always show the widget.

Reply:
What is the experience when the sign is `VIENNA` then? Does it change on every screen change? Seems like an issue someone will raise 🤔

Reply:
Yes, by default widget uses MUTCD style after initialization. Users can override that by using `SpeedLimitOptions` or by instantiating the widget themselves. Is there anything we can do about it?

Reply:
By the way, Drop-In UI has the same issue:

https://user-images.githubusercontent.com/2395284/188944472-2595cc0e-9b1d-4c19-be23-1d94090a6caa.mp4

Reply:
Considering that it's an issue on drop-in-ui is not a good reason to create the same issue in androidauto though. If we find a good fix to the problem in androidauto we can also fix drop-in-ui.

> Speed limit widget shows not only the speed limit, but also the current speed

But the speed also comes from the `LocationMatcherResult`. So there is not a speed without the speed limit sign type. There's no reason to show an empty speed limit sign with the wrong format 🤔 . In the video from drop-in-ui, it's showing the wrong speed limit sign, with no speed value and no speed limit.

Reply:
Think we should fix it here and cut a new issue for drop-in-ui

Reply:
> But the speed also comes from the `LocationMatcherResult`. So there is not a speed without the speed limit sign type. There's no reason to show an empty speed limit sign with the wrong format 🤔 .

No, `speedLimitSign` comes from `LocationMatcherResult.speedLimit`, which is null, when speed limit info is not available.

Reply:
Where else is it coming from? Am I blind on this pull request because I can only see it here. There is the `forcedSignFormat`, but that would be different because it is coming from something external

```
    private fun updateSpeed(locationMatcherResult: LocationMatcherResult) {
        val speedKmph =
            locationMatcherResult.enhancedLocation.speed / METERS_IN_KILOMETER * SECONDS_IN_HOUR
        val speedLimitOptions = mainCarContext.speedLimitOptions.value
        val signFormat = speedLimitOptions.forcedSignFormat
            ?: locationMatcherResult.speedLimit?.speedLimitSign
        val threshold = speedLimitOptions.warningThreshold
````

Reply:
It's definitely a user experience issue to show an empty, wrong format, speed limit sign. 

Many solutions will not start a session until they have to, because that triggers billing. In that case, it seems clear to me that the desired solution would be to not show the speed limit sign.

Do we agree here? Or do you want to move forward with an empty speed limit sign with the wrong format during initialization? 🤔

Reply:
As you can see, speed limit info has to be present in order to retrieve the speed limit sign:
```kotlin
val signFormat = speedLimitOptions.forcedSignFormat
    ?: locationMatcherResult.speedLimit?.speedLimitSign
```
Which means you can have speed info, but not speed limit info and therefore no speed limit sign. I don't think there is a way around it at the moment other than to split the widget into two parts and show current speed using a design, that does not depend on the speed limit sign type.